plugins/janus_lua.c

```diff
@@ -1246,6 +1250,7 @@ void janus_lua_create_session(janus_plugin_session *handle, int *error) {
 	lua_getglobal(t, "createSession");
 	lua_pushnumber(t, session->id);
 	lua_call(t, 1, 0);
+	lua_pop(lua_state,1);
```

Comment:
Space missing after the comma (coding style). May be the same in other lines, haven't checked.

Apart from that, why is `createSession` creating a thread, and not `init`, when both pass an argument and ignore the result? What's the rationale behind choosing whether to use the the state directly or creating a thread instead?